dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/EventDAO.java

```diff
@@ -109,24 +114,52 @@ private PredicateBuilder initiatorPredicateBuilder(long initiatorUserId) {
 
     private PredicateBuilder accessPredicateBuilder(User loggedInUser) {
         return (cb, event) -> {
-            // Calculate a predicate representing public access.
-            // Currently, this is all events that don't refer to a "categorizer" organization.
-            Join<Event, Organization> organization = event.join("organization", JoinType.LEFT);
-            Predicate publicPredicate = cb.or(cb.isFalse(organization.get("categorizer")), cb.isNull(organization));
-            // Calculate a predicate representing extra access due to the user's identity.
-            Predicate extraPredicate;
             if (loggedInUser != null && (loggedInUser.getIsAdmin() || loggedInUser.isCurator())) {
-                // Admin or curator, able to access everything.
-                extraPredicate = cb.conjunction();
-            } else if (loggedInUser != null) {
-                // Logged in, able to access events that the user initiated.
-                extraPredicate = event.get("initiatorUser").in(loggedInUser.getId());
+                // An admin or curator is able to access everything.
+                return cb.conjunction();
+            }
+            // Calculate a predicate representing public access, which, currently, is all events that:
+            // 1) Either:
+            // a) Don't refer to an entry
+            // b) Refer to a published entry, or
+            // c) Are of type PUBLISH_ENTRY or UNPUBLISH_ENTRY or COLLECTION-related events
```

Comment:
Similarly, can you see unpublished for all entries, or just ones you have starred?

Reply:
See above.